code/go/0chain.net/smartcontract/zcnsc/authorizer_test.go

```diff
@@ -140,26 +140,27 @@ func Test_Basic_ShouldSaveGlobalNode(t *testing.T) {
 	require.Equal(t, state.Balance(100*1e10), globalNode.MinStakeAmount)
 }
 
-func TestShould_Fail_If_TransactionValue_Less_Then_GlobalNode_MinStake(t *testing.T) {
-	ctx := MakeMockStateContext()
-	sc := CreateZCNSmartContract()
-
-	client := defaultAuthorizer + time.Now().String()
-	input := CreateAuthorizerParamPayload(client)
-
-	tr := CreateAddAuthorizerTransaction(client, ctx)
-	tr.Value = 99
-
-	node := CreateSmartContractGlobalNode()
-	node.MinStakeAmount = state.Balance(100 * 1e10)
-	err := node.Save(ctx)
-	require.NoError(t, err)
-
-	resp, err := sc.AddAuthorizer(tr, input, ctx)
-	require.Error(t, err)
-	require.Empty(t, resp)
-	require.Contains(t, err.Error(), "min stake amount")
-}
+// INFO: Temporarily disabled as fees will be updated in global fee update
```

Comment:
it's not fee, it's a min stake for a pool

Reply:
Removed this. Now, the staking (or lock) is a standalone process